crates/air-lib/air-parser/src/ast/instructions/traits.rs

```diff
@@ -175,3 +175,47 @@ impl fmt::Display for New<'_> {
         write!(f, "new {}", self.argument)
     }
 }
+
+macro_rules! peer_id_error_logable {
+    ($($t:ty),+) => {
+        $(
+            impl PeerIDErrorLogable for $t {
+                fn log_errors_with_peer_id(&self) -> bool {
```

Comment:
Methods so small should be decorated with `#[inline]`, as it is called in the other module.